py/desispec/io/meta.py

```diff
@@ -782,7 +782,10 @@ def rawdata_root():
     Raises:
         KeyError: if these environment variables aren't set.
     """
-    return os.environ['DESI_SPECTRO_DATA']
+    if 'DESI_SPECTRO_DATA' in os.environ:
+        return os.environ['DESI_SPECTRO_DATA']
+    else:
+        return os.path.expandvars('$DESI_ROOT/spectro/data')
```

Comment:
Could we make this `os.path.join(os.environ['DESI_ROOT'], 'spectro', 'data')` to be more similar to the other code below?